server/router/rider.ts

```diff
@@ -8,6 +8,27 @@ const router = express.Router();
 AWS.config.update(config);
 const docClient = new AWS.DynamoDB.DocumentClient();
 
+type Rider = {
+  id: string,
+  firstName: string,
+  lastName: string,
+  phoneNumber: string,
+  email: string,
+  accessibilityNeeds: {
+    needsWheelchair: boolean,
+    hasCrutches: boolean,
+    needsAssistant: boolean,
+  },
+  description: string,
+  picture: string,
+  joinDate: number,
```

Comment:
Why is this a number? Should it be a string in some ISO date format?

Reply:
i was planning on storing it w/ a unix timestamp but i actually forgot about ISO so i'll change that